packages/react-core/src/components/Menu/MenuItemAction.tsx

```diff
@@ -45,24 +45,29 @@ const MenuItemActionBase: React.FunctionComponent<MenuItemActionProps> = ({
             onActionClick && onActionClick(event, itemId, actionId);
           };
           return (
-            <button
+            <div
               className={css(
                 styles.menuItemAction,
                 isFavorited !== null && 'pf-m-favorite',
                 isFavorited && styles.modifiers.favorited,
                 className
               )}
-              aria-label={ariaLabel}
-              onClick={onClickButton}
-              {...((isDisabled === true || isDisabledContext === true) && { disabled: true })}
-              ref={innerRef}
-              tabIndex={-1}
               {...props}
             >
-              <span className={css(styles.menuItemActionIcon)}>
-                {icon === 'favorites' || isFavorited !== null ? <StarIcon aria-hidden /> : icon}
-              </span>
-            </button>
+              <Button
+                aria-label={ariaLabel}
+                onClick={onClickButton}
+                ref={innerRef}
+                role="menuitem"
+                variant="plain"
+                tabIndex={-1}
+                isDisabled={isDisabled || isDisabledContext}
+              >
+                <span className={css(styles.menuItemActionIcon)}>
+                  {icon === 'favorites' || isFavorited !== null ? <StarIcon aria-hidden /> : icon}
+                </span>
```

Comment:
Since `menu-item-action` is being updated to use the `button` component, you shouldn't need this wrapper at all. 

```suggestion
                {icon === 'favorites' || isFavorited !== null ? <StarIcon aria-hidden /> : icon}
```